app/views/admin/neighbourhoods/new.html.erb

```diff
@@ -2,16 +2,6 @@
   const currentInfo = <%= @neighbourhood.to_json.html_safe %>
 </script>
 
-<h1>Quick Add</h1>
-<div id="js-ward-picker">
-  <div class="row">
-    <div class="col-md-6">
-      <div class="form-group">
-        <label class="form-control-label" for="ward-input">Quick add by ward code (WD19CD)</label>
-        <input class="form-control" id="ward-lookup" v-model="wardSearch" v-on:change="lookupWard()" placeholder="E01234567"></input>
-      </div>
-    </div>
-  </div>
+<p class="alert alert-danger" role="alert">Warning: neighbourhoods should not be created here and this page is only left as a placeholder!</p>
```

Comment:
Do we want to remove it? What breaks if we do not have a placeholder?

Reply:
Hmmm I did consider this but I think it's good to resolve the usual default Rails routes? The main usecase for it going forwards is for when we have custom geographies but that's a way off?